### PR TITLE
Sea tracking

### PR DESCRIPTION
**Pull request by liam-metke**
All harnesses in the jobs2 directory are directly copied from the jobs directory, but the names are changed to reflect the jobs2 naming scheme. Note all the sea.yaml files and CMakeLists.txt files are configured for the jobs2 variant of the harness.

### CODE REVIEW

File: seahorn/jobs2/byte_buf_eq_ignore_case2/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+sea_attach_bc_link(byte_buf_eq_ignore_case2)
+configure_file(sea.yaml sea.yaml @ONLY)
+sea_add_unsat_test(byte_buf_eq_ignore_case2)
+
+# fuzz
+sea_add_fuzz(byte_buf_eq_ignore_case2
+  aws_byte_buf_eq_ignore_case_harness2.c)
+
+# smack
+sea_add_smack(byte_buf_eq_ignore_case2 ${AWS_C_COMMON_ROOT}/source/byte_buf.c
```
Comment (priyasiddharth) [inline seahorn/jobs2/byte_buf_eq_ignore_case2/CMakeLists.txt:13]: This test should be applied to smack since it uses seahorn primitives

**Review by priyasiddharth — CHANGES_REQUESTED** [review]

File: seahorn/jobs2/byte_buf_eq_ignore_case2/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+add_executable(byte_buf_eq_ignore_case2
+  ${AWS_C_COMMON_ROOT}/source/byte_buf.c
+  aws_byte_buf_eq_ignore_case_harness2.c)
+sea_attach_bc_link(byte_buf_eq_ignore_case2)
+configure_file(sea.yaml sea.yaml @ONLY)
+sea_add_unsat_test(byte_buf_eq_ignore_case2)
+
+# fuzz
```
Comment (priyasiddharth) [inline seahorn/jobs2/byte_buf_eq_ignore_case2/CMakeLists.txt:8]: remove fuzz also. Not needed.

File: seahorn/jobs2/array_eq_c_str2/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+add_executable(array_eq_c_str2
+  ${AWS_C_COMMON_ROOT}/source/byte_buf.c
+  aws_array_eq_c_str_harness2.c)
+sea_attach_bc_link(array_eq_c_str2)
+
+configure_file(sea.yaml sea.yaml @ONLY)
+sea_add_unsat_test(array_eq_c_str2)
+
+sea_add_fuzz(array_eq_c_str2
```
Comment (priyasiddharth) [inline seahorn/jobs2/array_eq_c_str2/CMakeLists.txt:9]: remove fuzz -- not needed

File: seahorn/jobs2/byte_buf_eq_c_str2/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+add_executable(byte_buf_eq_c_str2
+  ${AWS_C_COMMON_ROOT}/source/byte_buf.c
+  aws_byte_buf_eq_c_str_harness2.c)
+sea_attach_bc_link(byte_buf_eq_c_str2)
+configure_file(sea.yaml sea.yaml @ONLY)
+sea_add_unsat_test(byte_buf_eq_c_str2)
+
+sea_add_fuzz(byte_buf_eq_c_str2
```
Comment (priyasiddharth) [inline seahorn/jobs2/byte_buf_eq_c_str2/CMakeLists.txt:8]: remove fuzz

**Review by priyasiddharth — CHANGES_REQUESTED** [review]

File: seahorn/jobs2/byte_buf_eq_ignore_case2/CMakeLists.txt (hunk 2 above)
Comment (liam-metke) [reply seahorn/jobs2/byte_buf_eq_ignore_case2/CMakeLists.txt]: Removed fuzz

File: seahorn/jobs2/byte_buf_eq_c_str2/CMakeLists.txt (hunk 4 above)
Comment (liam-metke) [reply seahorn/jobs2/byte_buf_eq_c_str2/CMakeLists.txt]: Removed fuzz

File: seahorn/jobs2/array_eq_c_str2/CMakeLists.txt (hunk 3 above)
Comment (liam-metke) [reply seahorn/jobs2/array_eq_c_str2/CMakeLists.txt]: Removed fuzz

**Comment by priyasiddharth** [comment]
I am accepting the PR since the failing test has not been touched and I believe it is flaky.

**Pull request merged**